runtime/themes/fleet_dark.toml

```diff
@@ -76,25 +82,25 @@
 "ui.text" = "light" # .focus / .info
```

Comment:
**suggestion**: this could provide better highlighting for the pickers matching the picker of fleet.
```suggestion
"ui.text" = "light" # .info
"ui.text.focus" = { fg = "lightest", bg = "blue_accent" }
```

Reply:
Could you provide a screenshot of what this looks like? I'm currently away from my laptop

Reply:
Sure think, sorry. 

Here's how the picker looks currently:

![Screenshot 2022-12-04 at 22 33 18](https://user-images.githubusercontent.com/15747583/205517010-a744af98-d260-445d-b975-f083d9186896.png)

Here's how it would look like with the change above (which is actually incorrect):

![Screenshot 2022-12-04 at 22 33 56](https://user-images.githubusercontent.com/15747583/205517021-7da00819-818c-4dcb-a4f0-8a8914aab0fd.png)

The correct highlight color should be `#204474` which would look like this:

![Screenshot 2022-12-04 at 22 38 06](https://user-images.githubusercontent.com/15747583/205517049-377412d6-11ff-4dfa-a15e-394ca8a60de5.png)

And that matches fleet:

<img width="1552" alt="Screenshot 2022-12-04 at 22 38 16" src="https://user-images.githubusercontent.com/15747583/205517058-918dd7ac-5969-4e32-aded-537830e5cf0f.png">

I guess the same should be used for `ui.menu.selected`, currently looking like this:

<img width="420" alt="Screenshot 2022-12-04 at 22 44 32" src="https://user-images.githubusercontent.com/15747583/205517253-54994de2-e27f-4531-8b71-f7d6cecb23f6.png">

versus (in Fleet):

<img width="420" alt="Screenshot 2022-12-04 at 22 44 32" src="https://user-images.githubusercontent.com/15747583/205517267-e95aa3d0-bd4a-420f-bed0-7c784e4639ff.png">

I.e., maybe this would work:

```toml
...
"ui.menu.selected" = { fg = "lightest", bg = "dark_blue_accent" } # .selected
"ui.text.focus" = { fg = "lightest", bg = "dark_blue_accent" }

...

```

Reply:
Looks great! Good job. Can we reuse the "selected" color or is that too dark?

Reply:
If we are to follow Fleet strictly it uses 2 different color, the one configured right now for `selected` for selections and the one above for the menues.

Reply:
Alright. Maybe name it "focus" instead of accent, as accent colors are much brighter than anything else?

Reply:
Yes, sounds good 👍